Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -244,7 +244,10 @@ - (void)launchApplicationAndExecuteTestsWithParser:(BPTreeParser *)parser andCom
         hostBundleId = [SimulatorHelper bundleIdForPath:self.config.testRunnerAppPath];
     }
     // Create the environment for the host application
-    NSDictionary *argsAndEnv = [BPUtils buildArgsAndEnvironmentWith:self.config.schemePath];
+
+    NSMutableDictionary *argsAndEnv = [[NSMutableDictionary alloc] init];
+    argsAndEnv[@"args"] = self.config.commandLineArguments ?: [[NSArray alloc] init];
```

Comment:
Shorthand for [[NSArray alloc] init]: @[]

Also it seems odd that the dictionary is mutable but the array is not. Is that correct? I can't find any examples of mutation in this set of changes.